scpd/lms/static/sass/partials/base/variable_overrides/images.scss

```diff
@@ -0,0 +1 @@
+$homepage-bg-image: url('/static/scpd/images/stanford-pano.jpg');
```

Comment:
Should we make the filename a little less stanford-specific, like just `panoramic.jpg`?